components/checkbox/README.md

```diff
@@ -8,66 +8,67 @@ author: orinevares
+Based on [research](https://designnotes.blog.gov.uk/2016/11/30/weve-updated-the-radios-and-checkboxes-on-gov-uk/) done by Gov.UK.
 
 ## Behaviour
 1. Users should be able to tap or click on either the checkbox or the label to select the option.
+2. Do not preselect options, this signals a biased result.
 
 ## Accessibility
-This checkbox has been built according to WCAG 2.0 AA standards and has been successfully tested for the following:
-1.	Perceivable
-* Alternative text is provided for non-text content (WCAG 1.1.1)
-*	Content can be presented in different ways without losing information or structure (WCAG 1.3)
-*	Colour is not used as the only visual means of conveying information (WCAG 1.4.1)
-*	Visual presentation of text and images has a contrast ratio of at least 4.5:1 (WCAG 1.4.3)
-*	Text can be resized without assistive technology up to 200% without loss of content or functionality (WCAG 1.4.4)
-
-2.	Operable
-*	All functionality of the content is operable through a keyboard interface (WCAG 2.1.1)
-*	A mechanism is available to bypass blocks of content that are repeated on multiple web pages (WCAG 2.4.1)
-*	Headings and Labels describe topic or purpose (WCAG 2.4.6)
-*	Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible (WCAG  2.4.7)
-
-3.	Understandable
-*	The default human language of each web page can be programmatically determined (WCAG 3.1.1)
-*	Navigational mechanisms that are repeated on multiple web pages occur in the same relative order each time they are repeated (WCAG 3.2.3)
-
-4.	Robust
-*	For all user interface components, the name and role can be programmatically determined (WCAG 4.1.2)
-
-## Design with This Component
+This checkbox has been built according to [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards and all government services should strive to meet this level.  This component successfully includes the following accessibility features:
+
+### Screenreaders
+* Proper ARIA and ALT labels are included
+* Language tag has been set to english
+* List items and checkbox status are labelled properly
+
+### Colour Contrast
+* Contrast ratios are at least 7:1 for normal text and 4.5:1 for large text
+
+### Tab Navigation
+* Supports tab navigation for use without a mouse
+
+### Learn More
+* Contacts
+* Online Tools
+* Online Resources
+* Gov Resources
+
+## Prototype with This Component
 Download and use the B.C. government Design System library to help build prototypes and wireframes for your website or application.
 
 The library is constantly updated and currently available in the following tools:
 
-•	Sketch: [https://sketch.cloud/s/Q0bkG] 
-
-This library is updated on a regular basis; by subscribing to the design library, you’ll be informed of updates to components and new additions.
+*	[Sketch](https://sketch.cloud/s/Q0bkG)
+* [UxPin and others](https://sketch.cloud/s/Q0bkG)
```

Comment:
I like this like to UXPin and others, but wonder if it could be confusing to just link to the same Sketch library. Can we maybe create a page explaining how to integrate sketch files into UXpin and other tools, and which tools are currently compatible?

Reply:
Ya I agree we need an explanation page. The UxPin link is really to just convey an initial understanding that the sketch library works with other tools. I'll take out the link for now until we have a page or process outlining how this is done.